main.cpp

```diff
@@ -52,84 +52,109 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: bulldozer
+//  action 1: the blade goes up
+blade.up();
 
-//  action 2:
+//  action 2: the blade goes down
+blade.down();
 
-//  action 3:
+//  action 3:the bulldozer drives forward
+drive.forward();
 
 //  2)
-//  Noun:
-//  action 1:
+//  Noun:crane
+//  action 1:rotate to the left
+rotate.left();
 
-//  action 2:
+//  action 2:rotate to the right
+rotate.right();
 
-//  action 3:
+//  action 3: raise the pulley and hook slowly
+raise.slow();
 
 //  3)
-//  Noun:
-//  action 1:
+//  Noun:welder
+//  action 1:joins bars
+welder.join();
 
-//  action 2:
+//  action 2:cuts bars
+welder.cut();
 
-//  action 3:
+//  action 3:pickup the pieces
+welder.pickup();
 
 //  4)
-//  Noun:
-//  action 1:
+//  Noun:dumpTruck
+//  action 1:drives forward
+dumpTruck.forward();
 
-//  action 2:
+//  action 2: drives in reverse
+dumpTruck.reverse();
 
-//  action 3:
+//  action 3: dumpTruck drops off load
+dumpTruck.droffOff();
 
 //  5)
-//  Noun:
-//  action 1:
+//  Noun:hammer
+//  action 1:pounds a nail
+hammer.pound();
 
-//  action 2:
-
-//  action 3:
+//  action 2:pulls out a nail
+hammer.pull();
+//  action 3: falls to ground
+hammer.falls();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun:mechanic
+//  action 1:fixes car
+mechanic.fixesCar();
+//  action 2:dissasembles engine
+mechanic.disassembles();
 
-//  action 3:
+//  action 3:assembles widget
+mechanic.assembles();
 
 //  7)
-//  Noun:
-//  action 1:
+//  Noun:screwdriver    
+//  action 1:Drive in a screw
+screwdriver.drive();
 
-//  action 2:
+//  action 2:Remove a screw
+screwdriver.remove();
 
-//  action 3:
+//  action 3:Pry off lock
+screwdriver.pry();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.